src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/helper/DocumentHelper.java

```diff
@@ -615,10 +612,11 @@ public <T> T deepCopyArray(T object, TypeReference<T> typeReference) {
         }
     }
 
-    public String formatAddressForLetterPrinting(Map<String, Object> address) {
+    public String formatAddressForLetterPrinting(Map<String, Object> address, boolean isInternational) {
```

Comment:
There seems to be three instances of this formatAddressForLetterPrinting. Is it possible to extract it out to a separate class e.g. LetterAddressFormatter

Reply:
This refactor can be done in Tech-debt ticket.